src/components/dictionaryConcepts/containers/CreateConcept.jsx

```diff
@@ -93,9 +93,9 @@ export class CreateConcept extends Component {
     const isAddedConcept = addedConcept.length;
     if (isNewConcept && isAddedConcept) {
       setTimeout(() => {
-        notify.show('concept successfully created', 'success', 3000);
```

Comment:
How does this change disable the create button?

Reply:
It doesn't. The reason I did change that was to redirect to the concepts page first, then alert the success message.

Reply:
In other words, is this change related to the ticket?

Reply:
Yes, it is. I realized that props were being updated faster than the given timeout hence the button is being re-enabled before the page redirects.